src/__tests__/index.test.ts

```diff
@@ -138,6 +138,11 @@ const vectors: Array<TestVector> = [
       },
     ],
   },
+  {
+    name: 'XMR',
+    coinType: 128,
+    passingVectors: [{ text: 'AXaXZjZGA3qhQRTCsyG5uFKr9HeShgVhTF', hex: '17ad5cac596a1ef6c18ac1746dfd304f93964354b5' }],
```

Comment:
Where did you get this test address? Your reference https://web.getmonero.org/resources/moneropedia/address.html says `A raw Monero address is a set of 95 characters starting with a '4'. The Monero donation address, for instance, is 888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnjyPbb3iQ1YBRk1UXcdRsiKc9dhwMVgN5S9cQUiyoogDavup3H.` which looks quite different